CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
This is the version I have, it is what is put in the file by default. If someone has an earlier version and it works unchanged for them then definitely that number can be lowered to that number.